tools/trace_analyzer_tool.cc

```diff
@@ -1162,15 +1162,16 @@ Status TraceAnalyzer::ReProcessing() {
 
 // End the processing, print the requested results
 Status TraceAnalyzer::EndProcessing() {
+  Status s;
   if (trace_sequence_f_) {
-    trace_sequence_f_->Close();
+    s = trace_sequence_f_->Close();
   }
   if (FLAGS_no_print) {
-    return Status::OK();
+    return s;
   }
   PrintStatistics();
   CloseOutputFiles();
```

Comment:
Note that CloseOutputFiles does a bunch of file Close without checking the status returned.  You are probably just never hitting a test here?

Reply:
Let me update the status check in CloseOutputFiles(). Yes, testing does not hit that case.